File: CRM/src/app/shared/eHospital_DongNai_A_Config/Sys_HistoryDefines.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_HistoryDefines } from './Sys_HistoryDefines.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_HistoryDefinesService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'HistoryDefine_Id', 'DataGroup', 'TableName', 'Insert_Enabled', 'Update_Enabled', 'Delete_Enabled', 'Description'];

    List: Sys_HistoryDefines[] | undefined;
    ListFilter: Sys_HistoryDefines[] | undefined;
    FormData!: Sys_HistoryDefines;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_HistoryDefines";
    }
}

File: Service_eHospital_DongNai_A_Config/Implement/Sys_HistoryDefinesService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_HistoryDefinesService : BaseService<Sys_HistoryDefines, ISys_HistoryDefinesRepository>
    , ISys_HistoryDefinesService
    {
    private readonly ISys_HistoryDefinesRepository _Sys_HistoryDefinesRepository;
    public Sys_HistoryDefinesService(ISys_HistoryDefinesRepository Sys_HistoryDefinesRepository) : base(Sys_HistoryDefinesRepository)
    {
    _Sys_HistoryDefinesRepository = Sys_HistoryDefinesRepository;
    }
    }
    }

